warehouse/query-core/src/main/java/datawave/query/iterator/QueryOptions.java

```diff
@@ -1938,7 +1939,7 @@ public Iterator<Entry<Key,Document>> getPostProcessingChain(Iterator<Entry<Key,D
                             ((ConfiguredPredicate) p).configure(options);
                         }
 
-                        tforms = QueryIterator.statelessFilter(tforms, p);
+                        tforms = QueryIterator.statelessFilter(tforms, p, new QuerySpan(getStatsdClient()));
```

Comment:
This query span needs to be passed from the query iterator.  Otherwise these metrics will never get passed back to the client/webserver